index.js

```diff
@@ -121,6 +123,9 @@ Toolkit.run(async tools => {
     const remoteRepo = `https://${process.env.GITHUB_ACTOR}:${process.env.GITHUB_TOKEN}@github.com/${process.env.GITHUB_REPOSITORY}.git`
     if (process.env['INPUT_SKIP-TAG'] !== 'true') {
       await tools.runInWorkspace('git', ['tag', newVersion])
+      if(tagCommitMessage) {
+        await tools.runInWorkspace('git', ['tag', newVersion, `${newVersion}^{}`, '-f', '-m', `"${tagCommitMessage.replace(/{{version}}/g, newVersion)}"`])
```

Comment:
@vodlogic  what git command does that produce in your tests?

Reply:
git tag v8.2.6 v8.2.6\^{} -f -m "new tag message"